pr.js

```diff
@@ -41,19 +41,36 @@ function getReleaseType(pr, config) {
 
 // Extracts the release notes from the PR body.
 function getReleaseNotes(pr, config) {
-    let notes = ''
+    let notes = new Array();
+
     if (pr.body !== null && pr.body !== '') {
-        const matches = pr.body.match(config.releaseNotesRegex)
-        if (matches !== null && matches.length > 1) {
-            notes = matches[1].trim()
+        let lines = pr.body.split(/\r?\n/);
+        let withinNotes = config.releaseNotesPrefixPattern === undefined;
+        let firstLine = 0;
+        let lastLine = lines.length;
+
+        for (let i=0; i<lines.length; i++) {
```

Comment:
nit: please pad `=` and `<` here with spaces

Reply:
Fixed.